sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -9,6 +9,11 @@
 - Updated all long-running operation client methods to a new pattern. This affects `StartAnalyzeDocument`, `StartAnalyzeDocumentFromUri`, `StartBuildModel`, `StartCopyModelTo`, and `StartCreateComposedModel` methods. Changes are:
   - Removed the "Start" prefix. For example, `StartAnalyzeDocument` was renamed to `AnalyzeDocument`.
   - Added a new required parameter: `waitUntil`. It specifies whether the operation should run to completion before returning or not, removing the need to call `WaitForCompletion` in most scenarios.
+- Renamed `DocumentModel` to `DocumentModelDetails`.
+- Renamed `ModelOperation` to `DocumentModelOperationDetails`.
+- Renamed `ModelOperationInfo` to `DocumentModelOperationSummary`.
+- Renamed `AccountProperties` to `ResourceDetails`.
```

Comment:
Is `ResourceDetails` too generic? Is `FormRecognizerResourceDetails` too long?

Reply:
We have not added FormRecognizer prefix to any other names so it is not the convention we have followed till now.